packages/editor/src/lib/options.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Options for configuring tldraw. For defaults, see {@link defaultTldrawOptions}.
+ *
+ * @example
+ * ```tsx
+ * const options: Partial<TldrawOptions> = {
+ *     maxPages: 3,
+ *     maxShapesPerPage: 1000,
+ * }
+ *
+ * function MyTldrawComponent() {
+ *     return <Tldraw options={options} />
+ * }
+ * ```
+ *
+ * @public
+ */
+export interface TldrawOptions {
+	readonly maxShapesPerPage: number
+	readonly maxPages: number
+	readonly animationMediumMs: number
+	readonly followChaseViewportSnap: number
+	readonly doubleClickDurationMs: number
+	readonly multiClickDurationMs: number
+	readonly coarseDragDistanceSquared: number
+	readonly dragDistanceSquared: number
+	readonly defaultSvgPadding: number
+	readonly cameraSlideFriction: number
+	readonly gridSteps: readonly { min: number; mid: number; step: number }[]
```

Comment:
i think if we wanna be careful any nested properties should be `readonly` too
```suggestion
	readonly gridSteps: readonly { 
	  readonly min: number
	  readonly mid: number
	  readonly step: number
	}[]
```